builder/build.sh

```diff
@@ -46,11 +46,15 @@ fetch_r_source() {
   elif [ "${1}" = devel ]; then
     # Download the daily tarball of R devel
     wget -q https://stat.ethz.ch/R/daily/R-devel.tar.gz -O /tmp/R-devel.tar.gz
+  elif [ "${1}" = "next" ]; then
+    wget -q https://cran.r-project.org/src/base-prerelease/R-latest.tar.gz -O /tmp/R-next.tar.gz
   else
     wget -q "${CRAN}/src/base/R-`echo ${1}| awk 'BEGIN {FS="."} {print $1}'`/R-${1}.tar.gz" -O /tmp/R-${1}.tar.gz
   fi
   echo "Extracting R-${1}"
   tar xf /tmp/R-${1}.tar.gz -C /tmp
+  dirname=`tar tzvf /tmp/R-next.tar.gz | head -1 | awk '{ print $NF }' | cut -d/ -f1`
+  mv /tmp/${dirname} /tmp/R-${1}
```

Comment:
Seems like this will cause issues if the R version isn't `next`, and the tarball name is different. Should we only do it when the version is `next`?

Besides this, the changes look great to me though.

Reply:
Oh, right, good catch, silly me. Just fixed it. This time I checked the over versions as well.